functions/start-instance-test-access.ts

```diff
@@ -3,15 +3,14 @@ import * as access from 'wildebeest/backend/src/access'
 import * as errors from 'wildebeest/backend/src/errors'
 import { parse } from 'cookie'
 import type { ContextData } from 'wildebeest/backend/src/types/context'
-import type { InstanceConfig } from 'wildebeest/backend/src/config'
-import { configure } from 'wildebeest/backend/src/config'
 
-export const onRequestGet: PagesFunction<Env, any, ContextData> = async ({ request, data, env }) => {
-	return handleGetRequest(env.DATABASE, request)
+export const onRequestGet: PagesFunction<Env, any, ContextData> = async ({ request, env }) => {
+	return handleGetRequest(env, request)
 }
 
 // Route to test if Access has been configured properly
-export async function handleGetRequest(db: D1Database, request: Request): Promise<Response> {
+export async function handleGetRequest(env: Env, request: Request): Promise<Response> {
```

Comment:
why using Env instead of D1Database? that makes the handler harder to test

Reply:
> Why does it make it harder to test?
Because to test the handler we don't need to construct an env object, we can only pass a database.

Reply:
> Because to test the handler we don't need to construct an env object, we can only pass a database.
I am not sure I get what's harder about that, instead of passing `d1` you'd be passing `{ d1 } as Env`, doesn't seem much harder to me 😕 

But anyways the function does use the `env` so passing it is necessary regardless, isn't it?

Or again, am I missing something and an `env` object is globally available or something like that? (that in my opinion would make the function harder to test as you'd need to set the value before and after calling the function)

Reply:
Oh I see what you mean now. This endpoint isn't used anymore, that's why.

Reply:
can you please remove it in a followup PR?